openedx/core/djangoapps/dark_lang/tests.py

```diff
@@ -235,11 +232,17 @@ def test_partial_match_es_la(self, latin_america_code):
             self.process_middleware_request(accept=b'{};q=1.0, pt;q=0.5'.format(latin_america_code))  # pylint:disable=no-member
         )
 
-    def assert_session_lang_equals(self, value, session):
+    def _set_client_cookie_language(self, cookie_language):
+        """
+        Set the cookie language in the Client
+        """
+        self.client.cookies[settings.LANGUAGE_COOKIE_NAME] = cookie_language
```

Comment:
I have kept this as it is because a similar implementation was used with sessions. And the place where `set_if_set` was used, i used again with cookies.